lib/geocoder/lookups/olleh.rb

```diff
@@ -203,11 +225,11 @@ def query_url_params(query)
           newAddr: Olleh.new_addr_types[query.options[:new_addr_type]] || 0,
           isJibun: Olleh.include_jibun[query.options[:include_jibun]] || 0
        }
-      when "addr_step_search"
+      when 'addr_step_search'
         hash = {
           l_Code: query.options[:l_code]
         }
-      when "addr_nearest_position_search"
+      when 'addr_nearest_position_search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.